examples/producer_consumer.py

```diff
@@ -32,6 +32,7 @@
 
 loop = asyncio.get_event_loop()
 queue = asyncio.Queue(loop=loop)
-asyncio.ensure_future(produce(queue, 10), loop=loop)
```

Comment:
Yes, the idea here is to run the producer in a background task while waiting for the consumer to complete. However, I'm not opposed to this change if you think it makes the example more understandable. As an alternative, it is possible to make the consumer task more explicit:
```
loop = asyncio.get_event_loop()
queue = asyncio.Queue(loop=loop)
producer_task = loop.create_task(produce(queue, 10))
loop.run_until_complete(consume(queue))
assert producer_task.done()
loop.close()
```
In any case, it's probably good to [discourage the fire and forget approach](https://github.com/python/asyncio/issues/399) for the consumer task.